tests/conftest.py

```diff
@@ -6,20 +6,27 @@
 
 
 import pytest
+import numpy as np
 
 
 from pymagicc import MAGICC6, MAGICC7
+from pymagicc.io import MAGICCData
 
 
 MAGICC6_DIR = pkg_resources.resource_filename("pymagicc", "MAGICC6/run")
 TEST_DATA_DIR = join(dirname(__file__), "test_data")
 TEST_OUT_DIR = join(TEST_DATA_DIR, "out_dir")
 
+EXPECTED_FILES_DIR = join(TEST_DATA_DIR, "expected_files")
+
 
 def pytest_addoption(parser):
     parser.addoption(
         "--skip-slow", action="store_true", default=False, help="skip any slow tests"
     )
+    parser.addoption(
```

Comment:
What's this for? When MAGICC7 output changes somehow?

Reply:
In case we need to reformat the output files (cause we realise there's a mistake) and we don't want to do it by hand. The workflow here is basically: 

1. write an output file
1. look at it
1. if it's correct leave it, if not, update the code and re-run the relevant test with the `--update-expected-file` flag and repeat from step 1

Reply:
Ah, ok i get it now ...